karma.conf.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(config) {
+  config.set({
+
+    basePath: '.',
+
+    frameworks: ['browserify', 'jasmine'],
+
+    browserify: {
+      debug: true,
+      transform: [
+        ['babelify', {presets: ['es2015']}]
+      ]
+    },
+
+    files: [
+      'tests/*-spec.js'
+    ],
+
+    preprocessors: {
+      'tests/*-spec.js': 'browserify'
+    },
+
+    reporters: ['progress', 'dots'],
+
+    port: 9876,
```

Comment:
Do you need to set this? Can the default be used instead? Simpler I think to have less config